### DISCUSSION TITLE
Default Project is not getting connected in Local

### DISCUSSION THREAD

**Discussion by Vishruta-Patil** [supabase/supabase#12866] [Questions]
I'm currently working on an issue of Supabase repository for which I need to set up the project (dashboard) in the local. 

But on local, default project is not getting connected. Due to which I'm not able to work on the dashboard.

Permission is disabled as shown in the attached image. Also in the networks tab I checked, rest API is getting 404 error and query API is giving 400 error

Can someone please help me in this so that I continue working on the issue?

![supabase-error](https://user-images.githubusercontent.com/72292532/223465962-ef83a9a6-e444-4aa6-8df1-002827e4557c.PNG)

**Comment by ricardosikic** [comment]
same here, i can't connect my react app to local supabase config
